src/test/java/org/opensrp/web/rest/ClientFormResourceTest.java

```diff
@@ -224,9 +224,8 @@ public void testSearchForFormByFormVersionShouldReturnNextVersion() throws Excep
 
     @Test
     public void testAddClientFormWhenGivenJSON() throws Exception {
-        String formIdentifier = "opd/reg.json";
+        String formName = "opd/reg.json";
```

Comment:
The formName & formIdentifier are a bit different. I think we can rename the formName to form label but the formIdentifier still retains the purpose

Reply:
I've reverted this to how it was.